test/scripts/attest_fn_call.txtar

```diff
@@ -6,3 +6,4 @@ cp stdout out.json
 # [check] assert stderr and stdout expected values
 exec jq -r '.transitive_attested_function_call.claims.output | @base64d' out.json
 stdout 'Hello, World!'
+saveFile out.json attest-fn-call-out.json
```

Comment:
Here is the `saveFile` command in action. Recall, if no `saveDirectory` was specified then this will simply print a warning and continue on. This allows us to use this one script for both `TestAttestFnCall` and `TestAttestFnCallCombined`

Reply:
normally I'd say we should make a new testscript for a new test, but this seems like a totally fine concession given how complex the testscript is.